### ISSUE TITLE
iOS TopBar Right Button color is reset when popping

### ISSUE THREAD

**Issue by manuhook** [wix/react-native-navigation#7109]
## 🐛 Bug Report

The Topbar Right Button color is reset when popping

## To Reproduce

iOS : 
1/ On a screen (n°1) change the topbar right button color with mergeOptions
2/ Push a new screen (n°2)  and pop this screen
3/ The topbar right button color is reset on screen n°1

it's working fine on Android.

## Expected behavior

The Topbar Right Button color should keep the same color it had when the new screen was pushed when you go back to it.

## Actual Behavior

The topbar right button color is reset when popping. 
Note that the same is happening when cancelling a back gesture (you can see it at the end of the video)

## Your Environment

- React Native Navigation version: 7.13.0
- React Native version: 0.64.0
- Platform(s) (iOS, Android, or both?): iOS
- Device info (Simulator/Device? OS version? Debug/Release?): Simulator & Device, Debug & Release

## Reproducible Demo

Video :
https://imgur.com/a/d33fbnI

Thank you in advance 🙏

**Comment by manuhook** [comment]
I guess this issue its linked to this one : https://github.com/wix/react-native-navigation/issues/7090

**Comment by guyca** [comment]
Hey @manuhook 👋 
Does this issue reproduce in the playground app? If so, can you please fork and reproduce in the playground?

**Comment by manuhook** [comment]
I will try to send you a reproduction in the playground this week !

**Comment by manuhook** [comment]
Hi @guyca !

While trying to reproduce the problem on playground I realized that it occurred only when changing the color of the topbar button (through mergeOptions) with rightButtonColor / leftButtonColor.

I was able to work around the problem by change the color still with mergeOptions but using rightButtons: [{id: 'MY_ID', icon: 'MY_ICON', color: 'MY_COLOR'}].

I had read in another topic that rightButtonColor / leftButtonColor would be deprecated soon. Is this still planned? If yes maybe there is no point to fix this issue ?

Thanks
